docs/liquidators/manual.md

```diff
@@ -0,0 +1,162 @@
+# Manual Liquidations
+
+Running liquidations on Umee requires four main steps:
+
+- Preparing a liquidator account
+- Querying for eligible liquidation targets
+- Executing a liquidation transaction
+- Selling liquidation rewards for a profit
+
+This document contains notes on how to do these manually - a more sophisticated approach would be to automate one or more steps and monitor the results.
+
+## Preparing a Liquidator Account
+
+Any wallet you control can be used as a liquidator, as long as it has a pool of assets and tokens for gas on the relevant chains.
```

Comment:
Why talking about other chains? I think it is not relevant. Liquidation happens on Umee. So a liquidator has to have:
Umee account with tokens required for liquidation. Eg, to liquidate `100 AXL-USDC` and `50 ATOM`, the liquidator has to have at least:
1.  `100 AXL-USDC` and `50 ATOM`
2. `1UMEE` for transaction fees (variable).

Reply:
That's not the full picture - this manual covers the full operation because it is meant to help validators and other third parties get started.

Reply:
I know, but it reads that a user has to have other native tokens, which is not a hard requirements (again: user doesn't need to have native Osmo to liquidate AXL-USDC - this creates confusion when doing a first read). Hence my proposal to break it down, and firstly cover the basic usage, and in other section cover other recommending steps.

Reply:
I'll try to modify the first section to make it clear that those tokens are for gas. I do still like the chronological order (setup, query, liquidate, sell) for this guide, since it allows the user to follow step by step.

Reply:
Chronological order is good. But I would still separate it in two flows. It will make more sense.